ERCS/erc-xxxx.md

```diff
@@ -0,0 +1,111 @@
+description: Gateway attributes for cancellation, timeout, retry, dependencies, and execution control in cross-chain messaging.
+author: Ernesto García (@ernestognw), Kalman Lajko (@LajkoKalman), Valera Grinenko (@0xValera)
+discussions-to: TODO
+status: Draft
+type: Standards Track
+category: ERC
+created: 2024-XX-XX
+requires: 7786
+---
+
+## Abstract
+
+This ERC defines standard attributes for ERC-7786 cross-chain messaging gateways to enable consistent cancellation, timeout, retry, dependency, and execution control mechanisms across implementations. These attributes provide applications with predictable control over message lifecycle, ordering, and execution requirements.
+
+## Motivation
+
+ERC-7786 introduces an extensible attribute system for cross-chain messaging, but leaves attribute standardization to follow-up specifications. As cross-chain applications mature, consistent patterns for message control have emerged as essential requirements:
+
+1. **Cancellation**: Applications need to cancel pending messages due to changed conditions
+2. **Timeouts**: Automatic cancellation prevents indefinite pending states
+3. **Retry Logic**: Standardized failure handling improves reliability
+4. **Revert Behavior**: Consistent error semantics across gateways
+5. **Message Dependencies**: Ensuring correct ordering when messages must execute in sequence
+6. **Gas Requirements**: Preventing execution failures due to insufficient gas
+7. **Execution Timing**: Controlling when messages can be executed for scheduling and coordination
+
+Without standardized attributes, each gateway implements these features differently, fragmenting the ecosystem and requiring application-specific integration logic.
+
+## Specification
+
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD", "SHOULD NOT", "RECOMMENDED", "NOT RECOMMENDED", "MAY", and "OPTIONAL" in this document are to be interpreted as described in RFC 2119 and RFC 8174.
+
+### Standard Attributes
+
+This specification defines standard attributes for ERC-7786 cross-chain messaging gateways.
+
+Gateways MAY implement attributes independently. Gateways MUST validate the attribute's encoding for each attribute they implement and revert the transaction if the encoding is invalid.
+
+#### `cancellable(bool)`
+
+Indicates whether a message can be cancelled after submission. This attribute uses selector `0xde986d7f`, which represents the first 4 bytes of `keccak256("cancellable(bool)")`.
+
+The attribute value is encoded as an ABI-encoded boolean, and COULD default to `false` when not specified. When set to `true`, gateways MUST provide a cancellation mechanism to allow applications to cancel pending messages due to changed conditions or requirements.
+
+#### `timeout(uint256)`
+
+Specifies a timestamp after which the message is automatically cancelled. This attribute uses selector `0x08148f7a`, derived from the first 4 bytes of `keccak256("timeout(uint256)")`.
```

Comment:
> automatically cancelled
I would rephrase, "cannot be executed". ( with this phrasing there has to be an automatic cancellation mechanism). 

Alternatively, "implicitely cancelled". But